mkosi/__init__.py

```diff
@@ -1597,7 +1597,7 @@ def configure_dracut(args: CommandLineArguments, root: str) -> None:
     # efivarfs must be present in order to GPT root discovery work
     if args.esp_partno is not None:
         with open(os.path.join(dracut_dir, "30-mkosi-efivarfs.conf"), "w") as f:
-            f.write('add_drivers+=" efivarfs "\n')
+            f.write('[[ $(modinfo -k "$kernel" -F filename efivarfs 2>/dev/null) == /* ]] && drivers+=" efivarfs "\n')
```

Comment:
Shouldn't this remain `add_drivers+=" efivarfs "` instead of just `drivers+=" efivarfs "`? Currently this drops all other drivers, which would explain why Debian CIs fail. Otherwise this fix works, the `efivarfs`-related error is not present on systems where the `efivarfs` module is not a separate thing.

Reply:
yes... `add_drivers`